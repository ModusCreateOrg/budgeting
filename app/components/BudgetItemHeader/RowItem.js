// @flow
import * as React from 'react';
import styles from './style.scss';

type RowItemProps = {
  label: string,
  children: React.Element<any>,
};

const RowItem = ({ label, children }: RowItemProps) => (
  <div className={styles.rowItem}>
    <div className={styles.label}>{label} </div>
    <div className={styles.content}>{children} </div>
  </div>
);

export default RowItem;
