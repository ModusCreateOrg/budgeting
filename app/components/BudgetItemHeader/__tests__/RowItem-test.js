import React from 'react';
import renderer from 'react-test-renderer';
import RowItem from '../RowItem';

it('should render component correctly', () => {
  const label = 'Description';
  const value = 'New pair of shoes';

  const component = renderer.create(<RowItem label={label}> {value}</RowItem>).toJSON();
  expect(component).toMatchSnapshot();
});
