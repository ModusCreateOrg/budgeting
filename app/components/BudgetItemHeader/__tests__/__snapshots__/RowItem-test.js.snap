// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component correctly 1`] = `
<div
  className="rowItem"
>
  <div
    className="label"
  >
    Description
     
  </div>
  <div
    className="content"
  >
     
    New pair of shoes
     
  </div>
</div>
`;
